server/documents/modules/calendar.html.eco

```diff
@@ -339,6 +339,8 @@ themes      : ['Default']
     <div class="example" data-since="2.9.0">
       <h4 class="ui header">Custom format</h4>
       <p>You can entirely change the date format by using a token string formatter.</p>
+      <p>The format is applied to the calendar cells.</p>
+      <p>These same tokens can be used to format the date gotten through <a href="#/usage>behaviors</a>.</p>
```

Comment:
This is wrong. The format token is not only applied to the calendar cells but for several date outputs mentioned as tokens in the settings
![image](https://github.com/user-attachments/assets/132f5e82-0c0e-45b5-9714-6381977589ce)